### ISSUE TITLE
php keyword tab expansion only works from html

### ISSUE THREAD

**Issue by doginthehat** [atom/language-php#90]
This used to work fine on older version of atom but started happening recently.

When editing a php file, you can type the word `php`, then tab to expand to open/close tags.

It works when you're inside an html block, so:

```php
<div>
    php
</div>
```

will successfully expand to:

```php
<div>
    <?php  ?>
</div>
```

if you press tab after the `php` keyword.

However, if you're already inside a php block, eg:

```php
public function imafunction()
{
    php
} 
```

and press tab, it awkwardly results in this:

```php
public function imafunction()
{
    php  ?>
}
```

but it used to result in the much more useful

```php
public function imafunction()
{
    ?><?php
}
```

With the cursor placed correctly between `>` and `<`

**Comment by doginthehat** [comment]
* **Atom:**  1.0.2
* **OS:** Mac OS X 10.9.5
* **language-php:**  0.27.0

**Comment by Ingramz** [comment]
Related to #76 and #84

**Comment by frozzare** [comment]
When debugging this I notice that the `php` snippet in `.source.php` is trigger on
<img width="724" alt="skarmklipp 2015-07-14 20 12 40" src="https://cloud.githubusercontent.com/assets/14610/8680914/c0a9526e-2a64-11e5-8d12-92f783c17dc0.png">

But what I can se there are no `source php` css classes on any of the lines in the image. So #84 is not a correct fix for #76 since there are something else that trigger `php` snippet in `.source.php`.
